app/views/admin/index.scala.html

```diff
@@ -211,9 +223,8 @@ <h1>Activities</h1>
                                 <td>@LabelValidationTable.countYesterdayValidationsBasedOnResult(3) (@("%.0f".format(LabelValidationTable.countYesterdayValidationsBasedOnResult(3) * 100.0 / LabelValidationTable.countYesterdayValidations))%)</td>
                             </tr>
                         </table>
-                        <sup>*</sup>Number of users in the parentheses represent #users who have completed at least 1
-                        audit task (in the case of # of audit users) or have validated at least one label (in the case
-                        of # of validation users). Numbers outside the parenthesis represent total #users who opened
+                        <sup>*</sup>For cells with the format X (Y). The "Y" is the number of users who have completed at least 1
+                        audit task or have validated at least one label. The X is the total number of users who opened
                         the audit tool regardless of finishing a task.
```

Comment:
Should probably also say "who opened the audit or validation tool regardless of finishing a task."